server/cmd_scan_test.go

```diff
@@ -158,7 +174,24 @@ func TestSetScan(t *testing.T) {
 
 }
 
-func TestZSetScan(t *testing.T) {
+func TestSetScan(t *testing.T) {
+	c := getTestConn()
+	defer c.Close()
+
+	key := "scan_set"
+	c.Do("SADD", key, "a", "b")
+
+	if ay, err := goredis.Values(c.Do("SSCAN", key, "0")); err != nil {
+		t.Fatal(err)
+	} else if len(ay) != 2 {
+		t.Fatal(len(ay))
+	} else {
+		checkScanValues(t, ay[1], "a", "b")
+	}
+
```

Comment:
can we remove the empty line?

Reply:
Sure thing, I think it was there in the original code, but I'll remove it either way.

Thank you for responding so fast, appreciated as always! 💯